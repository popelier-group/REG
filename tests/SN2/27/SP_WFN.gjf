# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -1.41193200   -0.00002000    0.00006600
 H                 -1.08605900   -0.87535400   -0.54554700
 H                 -1.08605600    0.91017600   -0.48516100
 H                 -1.08615100   -0.03488600    1.03096400
 Br                 1.70551500    0.00000600   -0.00000400
 Cl                -3.27060800   -0.00000400   -0.00002000

frame.wfn
