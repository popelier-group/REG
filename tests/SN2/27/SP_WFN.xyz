
6

C        1.266617    0.038797    0.223287
H        1.114355   -0.511727   -0.695318
H        0.802781   -0.460841    1.063117
H        0.920466    1.059592    0.132412
Br       -1.802094   -0.055116   -0.317705
Cl        3.096234    0.094662    0.545866
