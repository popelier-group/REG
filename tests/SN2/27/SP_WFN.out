 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.27/Gau-75198.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.27/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/l1.exe PID=     75199.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -1.41193  -0.00002   0.00007 
 H                    -1.08606  -0.87535  -0.54555 
 H                    -1.08606   0.91018  -0.48516 
 H                    -1.08615  -0.03489   1.03096 
 Br                    1.70552   0.00001  0. 
 Cl                   -3.27061  0.        -0.00002 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.411932   -0.000020    0.000066
      2          1           0       -1.086059   -0.875354   -0.545547
      3          1           0       -1.086056    0.910176   -0.485161
      4          1           0       -1.086151   -0.034886    1.030964
      5         35           0        1.705515    0.000006   -0.000004
      6         17           0       -3.270608   -0.000004   -0.000020
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.081710   0.000000
     3  H    1.081710   1.786551   0.000000
     4  H    1.081711   1.786553   1.786555   0.000000
     5  Br   3.117447   2.976031   2.976013   2.976157   0.000000
     6  Cl   1.858676   2.415800   2.415793   2.415781   4.976123
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        1.266617    0.038797    0.223287
      2          1           0        1.114355   -0.511727   -0.695318
      3          1           0        0.802781   -0.460841    1.063117
      4          1           0        0.920466    1.059592    0.132412
      5         35           0       -1.802094   -0.055116   -0.317705
      6         17           0        3.096234    0.094662    0.545866
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         157.1087068           0.7987516           0.7987515
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       167.4971103306 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  2.68D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RB3LYP) =  -3074.40218030     A.U. after   11 cycles
            NFock= 11  Conv=0.72D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.55516-101.40979 -62.20820 -56.02600 -56.02593
 Alpha  occ. eigenvalues --  -56.02593 -10.08299  -9.32407  -8.41693  -7.08778
 Alpha  occ. eigenvalues --   -7.07931  -7.07931  -6.24001  -6.23965  -6.23965
 Alpha  occ. eigenvalues --   -2.35239  -2.35227  -2.35227  -2.35199  -2.35199
 Alpha  occ. eigenvalues --   -0.70936  -0.54316  -0.48839  -0.30727  -0.30727
 Alpha  occ. eigenvalues --   -0.27209  -0.17565  -0.17565  -0.05377  -0.05272
 Alpha  occ. eigenvalues --   -0.05272
 Alpha virt. eigenvalues --    0.10949   0.12344   0.14841   0.14841   0.17478
 Alpha virt. eigenvalues --    0.18342   0.19555   0.19555   0.20707   0.20707
 Alpha virt. eigenvalues --    0.21695   0.22445   0.22446   0.24807   0.28323
 Alpha virt. eigenvalues --    0.30734   0.30734   0.36696   0.40462   0.53521
 Alpha virt. eigenvalues --    0.53521   0.55325   0.58555   0.58555   0.60394
 Alpha virt. eigenvalues --    0.62074   0.62074   0.64755   0.76544   0.76544
 Alpha virt. eigenvalues --    0.79502   0.84297   0.87903   0.87904   0.89241
 Alpha virt. eigenvalues --    0.89242   0.95249   0.96789   0.96790   1.01079
 Alpha virt. eigenvalues --    1.01079   1.03382   1.08844   1.26339   1.26339
 Alpha virt. eigenvalues --    1.26762   1.56926   1.67632   1.67632   1.85056
 Alpha virt. eigenvalues --    1.85056   1.87203   2.09428   2.21435   2.21437
 Alpha virt. eigenvalues --    2.27474   2.27474   2.29441   2.45612   2.45613
 Alpha virt. eigenvalues --    2.56831   2.59500   2.64063   2.64064   2.67097
 Alpha virt. eigenvalues --    2.67098   3.11181   3.28989   3.87141   3.87141
 Alpha virt. eigenvalues --    4.39636   4.39638   4.40391   4.40392   4.51833
 Alpha virt. eigenvalues --    6.76374   7.80375   7.80376   7.85957   9.84049
 Alpha virt. eigenvalues --   24.03336  25.98726  25.98727  26.49083  48.07745
 Alpha virt. eigenvalues --  215.82625 290.01215 290.01215 290.062951020.83526
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.661002   0.403722   0.403719   0.403725  -0.058070  -0.160989
     2  H    0.403722   0.519767  -0.038270  -0.038272   0.000854  -0.040521
     3  H    0.403719  -0.038270   0.519767  -0.038271   0.000853  -0.040518
     4  H    0.403725  -0.038272  -0.038271   0.519785   0.000856  -0.040534
     5  Br  -0.058070   0.000854   0.000853   0.000856  35.922755   0.054633
     6  Cl  -0.160989  -0.040521  -0.040518  -0.040534   0.054633  17.231089
 Mulliken charges:
               1
     1  C   -0.653110
     2  H    0.192719
     3  H    0.192720
     4  H    0.192711
     5  Br  -0.921881
     6  Cl  -0.003160
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C   -0.074960
     5  Br  -0.921881
     6  Cl  -0.003160
 Electronic spatial extent (au):  <R**2>=           1198.6969
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              4.3717    Y=              0.1338    Z=              0.7707  Tot=              4.4411
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -74.8141   YY=            -44.4604   ZZ=            -45.3763
   XY=             -0.9289   XZ=             -5.3563   YZ=             -0.1638
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -19.9305   YY=             10.4232   ZZ=              9.5073
   XY=             -0.9289   XZ=             -5.3563   YZ=             -0.1638
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=            -44.6256  YYY=              0.0863  ZZZ=             -4.5059  XYY=             -8.5100
  XXY=             -0.8720  XXZ=             -4.8845  XZZ=             -9.0650  YZZ=             -1.1160
  YYZ=             -1.5756  XYZ=              0.0468
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=          -1556.4334 YYYY=            -77.3848 ZZZZ=           -117.9975 XXXY=            -24.8218
 XXXZ=           -142.7289 YYYX=            -19.7819 YYYZ=             -3.4934 ZZZX=           -118.9315
 ZZZY=             -4.0135 XXYY=           -249.3427 XXZZ=           -260.2266 YYZZ=            -32.6357
 XXYZ=             -1.7027 YYXZ=            -39.4985 ZZXY=             -7.6354
 N-N= 1.674971103306D+02 E-N=-7.688358600651D+03  KE= 3.070037704412D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE729\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\ZZALSPA2\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-1.411932,-0.00002,0.000066\H,0,-1.086059,-0.875354,-0.545547\H,0,
 -1.086056,0.910176,-0.485161\H,0,-1.086151,-0.034886,1.030964\Br,0,1.7
 05515,0.000006,-0.000004\Cl,0,-3.270608,-0.000004,-0.00002\\Version=ES
 64L-G16RevC.01\State=1-A\HF=-3074.4021803\RMSD=7.186e-09\Dipole=-1.747
 2823,-0.0000154,0.0000524\Quadrupole=-15.5410425,7.7705278,7.7705147,-
 0.0000475,-0.0001338,-0.0000003\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"



 IN THIS SHORT LIFE

 THAT ONLY LASTS AN HOUR

 HOW MUCH, HOW LITTLE, IS WITHIN OUR POWER....


 EMILY DICKINSON BOLTS OF MELODY NO. 521
 Job cpu time:       0 days  0 hours  0 minutes 14.8 seconds.
 Elapsed time:       0 days  0 hours  0 minutes  7.8 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:56:30 2020.
